Source/Core/Core/HW/EXI/BBA/BuiltIn.cpp

```diff
@@ -669,6 +685,37 @@ void CEXIETHERNET::BuiltInBBAInterface::ReadThreadHandler(CEXIETHERNET::BuiltInB
       }
     }
 
+    // Check for new UPnP client
```

Comment:
Could this be moved to a new function?  Just my take bu this function seems like it's doing a lot..